pythonwhois/net.py

```diff
@@ -46,15 +61,9 @@ def get_whois_raw(domain, server="", previous=None, rfc3490=True, never_cut=Fals
 			target_server = get_root_server(domain)
 	else:
 		target_server = server
-	if target_server == "whois.jprs.jp":
-		request_domain = "%s/e" % domain # Suppress Japanese output
-	elif domain.endswith(".de") and ( target_server == "whois.denic.de" or target_server == "de.whois-servers.net" ):
-		request_domain = "-T dn,ace %s" % domain # regional specific stuff
-	elif target_server == "whois.verisign-grs.com":
-		request_domain = "=%s" % domain # Avoid partial matches
-	else:
-		request_domain = domain
-	response = whois_request(request_domain, target_server)
+
+    response = whois_request(domain, target_server)
```

Comment:
I'd like to keep the 'special case' logic out of the `whois_request` function, so that that function just does an unmodified request. Perhaps something like `whois_request(apply_exceptions(domain), target_server)` would work here?